src/github/workflows/deploy.yml

```diff
@@ -327,9 +366,12 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
       run: |
+        sha="$(git rev-parse --short=7 HEAD)"
+
         gh release delete 'nightly' --repo "${GITHUB_REPOSITORY}" --cleanup-tag --yes || true
+        for i in artifacts/**/*; do mv "$i" "$(dirname ${i})/Agda-${sha}-$(basename ${i})"; done
         ls -R artifacts
-        gh release create 'nightly' artifacts/**/* --repo "${GITHUB_REPOSITORY}" --generate-notes --notes-start-tag v2.7.0 --prerelease --title "${{ needs.build.outputs.sha }}@master"
+        gh release create 'nightly' artifacts/**/* --repo "${GITHUB_REPOSITORY}" --generate-notes --notes-start-tag v2.7.0.1 --prerelease --title "${sha}@master"
```

Comment:
Do we have to change this every time we release? 🤔

Reply:
https://github.com/marketplace/actions/get-latest-tag Maybe this can be used?

Reply:
Yeah, but does Agda have consistent tags?

Reply:
> https://github.com/marketplace/actions/get-latest-tag Maybe this can be used?

This does not specify what it means by "latest tag", so one would have to try it out.  Latest by date certainly does not work.  What we want is the tag `v<ListNat>`where with the lexicographically greatest `<ListNat>`.

Reply:
> Do we have to change this every time we release? 🤔

Yes:
https://github.com/agda/agda/blob/2eba43d8f45c9d380c4e88d172d1533397544d36/notes/agda-releases.txt#L162-L165

Reply:
It appears that the following command works (at least, locally)

`git tag -l 'v*' --sort=-version:refname | grep -v -e '-rc[0-9]\+$' -e '-r[0-9]\+$' | head -n 1`

where
1. `--sort=-version:refname` sorts the list of tags in the descending order by the version numbers(*) (so `v2.10` comes before `v2.9`) and
2. the `grep` excludes release candidates. 

In the current repository, the above command gives `v2.7.0.1` as desired. 

(*) The git manual does not specify if this is exactly the lexicographical order.